uninstall/uninstall_linux.go

```diff
@@ -0,0 +1,90 @@
+package uninstall
+
+import (
+	"time"
+	"io/ioutil"
+	"os/exec"
+	"github.com/pufferpanel/pufferd/logging"
+	"syscall"
+)
+
+
+func StartProcess(){
+	killDaemon()
+	deleteFiles()
+	deleteUser()
+}
+
+func killDaemon(){
+	exec.Command("killall", "-15" "-u", "pufferd").Run()

```

Comment:
Okay, what about the killall -9 ?
And what if someone runs something through the pufferd user?